### PR TITLE
table doc

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**

### CODE REVIEW

File: gui/doc/chart.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ specify which trace you target.
 Indexed properties can have a default value (using the *property_name* syntax with
 no index) which is overridden by any specified indexed property.
 
+The _data_ property supported types are:
+- pandas Dataframe
```
Comment (FabienLelaquais) [inline gui/doc/chart.md:16]: I suspect would give details on the behavior of the control at this point.
How are those different input types processed...

File: gui/doc/table.md [hunk 2]
```diff
@@ -5,21 +5,58 @@ The table component supports 3 display modes:
    - _unpaginated_:  all rows and no page are shown (`show_all = True`).
    - _auto_loading_: the pages are loaded on demand depending on the visible area.
 
-## Details
+The _data_ property supported types are:
```
Comment (FabienLelaquais) [inline gui/doc/table.md:8]: I'm afraid you should give details on the behavior of the control at this point.
How are those different input types processed...

**Review by FabienLelaquais — APPROVED** [review]
Great job!

File: gui/doc/chart.md (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/chart.md:16]: described in ### Multiple charts with different lengths

**Pull request merged**